GLMakie/assets/shader/sprites.geom

```diff
@@ -164,11 +164,7 @@ void main(void)
     // Compute required amount of buffering
     float sprite_from_viewport_scale = 1.0 / viewport_from_sprite_scale;
     float bbox_buf = sprite_from_viewport_scale *
-                     (// Hack!! antialiasing is disabled for RECTANGLE==1 for now
-                      // because it's used for boxplots where the sprites are
-                      // long and skinny (violating assumption 1 above)
```

Comment:
I don't see a difference between master and this: (master | pr)
![Screenshot from 2023-04-30 10-19-31](https://user-images.githubusercontent.com/10947937/235343206-ec6613ad-3b03-405b-98a9-390683af82e6.png)![Screenshot from 2023-04-30 10-14-30](https://user-images.githubusercontent.com/10947937/235343211-8dd8d4b4-f0a0-4ea9-bcd8-c6a6bf50094b.png)

Reply:
This is coming from https://github.com/JuliaPlots/GLMakie.jl/pull/30, but without images I don't know what this actually fixed.